.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.18, 1.19]
+        go-version: [1.17, 1.18, 1.19]
```

Comment:
Question: Do we still support `go 1.17`? I think the min supported version is `go 1.18`.

Reply:
@qmuntal do you remember what was the rationale for also checking 1.17?  My memory got erased in the meantime :-)

Reply:
I was concerned about the go1.17 behavior commented by @shizhMSFT in https://github.com/veraison/go-cose/pull/116#discussion_r1032556161.

Reply:
thanks Quim.  I propose we drop 1.17 from the test matrix and make explicit in the README that we currently support 1.18 and 1.19.

@SteveLasker @roywill WDYT?

Reply:
gently prodding the @veraison/go-cose-maintainers ☝️

Reply:
I would suggest keeping `go 1.17` dropped.

Reply:
To minimize the support matrix, I'd vote for dropping and adding if someone opens a PR needing support, with a good explanation.

Reply:
@shizhMSFT @SteveLasker thanks for your input.  I have reverted the support matrix addition in [dd75bac](https://github.com/veraison/go-cose/pull/116/commits/dd75bac21583d93ea0f4f4c3bd6281ed0e5f083d)